### DISCUSSION TITLE
Funding?

### DISCUSSION THREAD

**Discussion by andrewdunndev** [wolveix/satisfactory-server#47] [Ideas]
Could you add a funding option so folks can tip you for enabling their gaming communities?

**Comment by wolveix** [comment]
I really appreciate you offering! While I'll maintain my stance that I do the work that I do for free because I love the community and enjoy the work itself, donations are of course massively appreciated! I've added a sponsor link to the side of the repo which links to my PayPal link account :) If there's a better way to handle donations, please let me know! And thank you!!

**Reply by wolveix** [reply]
@andrewgdunn @rawrgulmuffins I've added a sponsorship option to my GitHub profile. As a reminder, it's not expected nor required, though it's definitely appreciated!

**Comment by rawrgulmuffins** [comment]
I would sign up for a patreon page if you had one as a heads up.
